app/code/Magento/CatalogRule/Test/Unit/Model/Indexer/ReindexRuleProductPriceTest.php

```diff
@@ -104,8 +104,8 @@ public function testExecute()
             'product_id' => 100,
             'website_id' => 1,
             'customer_group_id' => 2,
-            'from_time' => mktime(0, 0, 0, date('m'), date('d') - 100),
-            'to_time' => mktime(0, 0, 0, date('m'), date('d') + 100),
+            'from_time' => mktime(0, 0, 0, (int)date('m'), (int)date('d') - 100),
```

Comment:
Could you explain why it's needed?

Reply:
I was asked to introduce `strict_types`. 
- `date(...): string` https://www.php.net/manual/en/function.date.php
- `mktime (int, int, int, int, int, int, int) : int`
Answer yourself :-)

Reply:
ah. ok. got it.